extensions/hibernate-search-elasticsearch/runtime/src/main/java/io/quarkus/hibernate/search/elasticsearch/runtime/HibernateSearchElasticsearchRecorder.java

```diff
@@ -124,7 +125,7 @@ private void contributeBackendBuildTimeProperties(BiConsumer<String, Object> pro
         private void contributeBackendRuntimeProperties(BiConsumer<String, Object> propertyCollector, String backendName,
                 ElasticsearchBackendRuntimeConfig elasticsearchBackendConfig) {
             addBackendConfig(propertyCollector, backendName, ElasticsearchBackendSettings.HOSTS,
-                    elasticsearchBackendConfig.hosts,
+                    elasticsearchBackendConfig.hosts.orElse(Collections.emptyList()),
```

Comment:
You should pass `null` here.

Like smallrye-config (apparently?), Hibernate Search makes a distinction between "no value" and "an empty list".
Passing an empty list here will explicitly request Hibernate Search to target no host at all, which does not make sense.

Reply:
This is fixed now.